### PR TITLE
Add schedule endpoint and models

### PR DESCRIPTION
**Pull request by chillymosh**
Added Schedule models
Added user method fetch_schedule
Added associated API calls in http
Added detail in client.fetch_channel docstring

### CODE REVIEW

File: twitchio/user.py [hunk 1]
```diff
@@ -693,6 +693,42 @@ async def create_prediction(
         )
         return Prediction(self._http, data[0])
 
+    async def fetch_schedule(
+        self,
+        segment_ids: List[str] = None,
+        start_time: str = None,
```
Comment (IAmTomahawkx) [inline twitchio/user.py:699]: this should really take a Datetime object and internally convert it

File: twitchio/user.py [hunk 2]
```diff
@@ -693,6 +693,42 @@ async def create_prediction(
...
+        utc_offset: str = None,
```
Comment (IAmTomahawkx) [inline twitchio/user.py:700]: this should take as an int

**Review by IAmTomahawkx — CHANGES_REQUESTED** [review]

File: twitchio/user.py (hunk 2 above)
Comment (chillymosh) [reply twitchio/user.py:700]: I'll change it to a int on input and convert to string on http request

File: twitchio/user.py (hunk 1 above)
Comment (chillymosh) [reply twitchio/user.py]: Changed to datetime and converts to RFC3339 string

**Review by IAmTomahawkx — APPROVED** [review]

**Pull request merged**